recipe/meta.yaml

```diff
@@ -109,43 +133,25 @@ outputs:
     {% set tests_to_skip = tests_to_skip + " or test_full_reimport" %}          # [ppc64le or s390x]
     {% set tests_to_skip = tests_to_skip + " or test_pep338" %}                 # [ppc64le or s390x]
     {% set tests_to_skip = tests_to_skip + " or test_no_typing_extensions" %}   # [ppc64le or s390x]
-    # 2022/5/5: E       AssertionError: (131, 1.54987431981169249551435343964035e-09, 4.0035173453529620614007210953362e-19, 'arcsinh')
-    # Flawed test when using MKL
-    # https://github.com/numpy/numpy/issues/16769
-    {% set tests_to_skip = tests_to_skip + " or test_overrides" %}  # [blas_impl == 'mkl']
-    # https://github.com/numpy/numpy/issues/15243
-    {% set tests_to_skip = tests_to_skip + " or test_loss_of_precision" %}  # [s390x or ppc64le]
-    # It should be fixed by https://github.com/numpy/numpy/issues/20426
-    # but it still fails on some platforms.
-    {% set tests_to_skip = tests_to_skip + " or test_new_policy" %}   # [s390x or ppc64le or (osx and arm64)]
-    # On osx-arm64: FAILED core/tests/test_limited_api.py::test_limited_api - subprocess.CalledProcessor
-    {% set tests_to_skip = tests_to_skip + " or test_limited_api" %}   # [osx and arm64]
-    # ?
-    {% set tests_to_skip = tests_to_skip + " or test_gcd_overflow" %}  # [s390x]
-    # ?
-    {% set tests_to_skip = tests_to_skip + " or test_herm_cases" %}  # [osx and arm64]
-    {% set tests_to_skip = tests_to_skip + " or test_generalized_herm_cases" %}  # [osx and arm64]
-    # ?
-    {% set tests_to_skip = tests_to_skip + " or test_partial_iteration_cleanup" %}  # [osx or win64]
 
     test:
       requires:
         - pip     # force installation or `test_api_importable` will fail
+        - cython >=0.29.30,<3.0
+        - wheel
         - setuptools <60.0.0
-        - pytest
         - hypothesis
-        - {{ compiler('c') }}  # [not osx]
-        - {{ compiler('cxx') }}  # [not osx]
-        - {{ compiler('fortran') }}  # [not osx]
-        - nomkl  # [x86 and blas_impl != 'mkl']
+        - pytest
+        - pytz
+        - pytest-cov
+        - cffi  # [py<310]
+        - mypy
+        - typing_extensions
       commands:
         - f2py -h
-        - python -c "import numpy; numpy.show_config()"
-        - export OPENBLAS_NUM_THREADS=1  # [unix]
-        - set OPENBLAS_NUM_THREADS=1  # [win]
-        - pytest -vvv --pyargs numpy -k "not ({{ tests_to_skip }})" --durations=0
+        - pip check
```

Comment:
What does `pip check` report if we drop the newly added test requirements?

Reply:
there unnecessary test requirements were removed